vispy/io/image.py

```diff
@@ -117,21 +116,26 @@ def read_png(filename):
     --------
     write_png, imread, imsave
     """
-    x = Reader(filename)
-    try:
-        alpha = x.asDirect()[3]['alpha']
-        if alpha:
-            y = x.asRGBA8()[2]
-            n = 4
-        else:
-            y = x.asRGB8()[2]
-            n = 3
-        y = np.array([yy for yy in y], np.uint8)
-    finally:
-        x.file.close()
-    y.shape = (y.shape[0], y.shape[1] // n, n)
-    return y
-
+    imageio, PIL = _check_img_lib()
```

Comment:
What do you think about a try/except ImportError with `import PIL` and changing the error message to say Pillow?

Reply:
Okay, will do.